### ISSUE TITLE
Large intensities at beginning of curve cause trouble

### ISSUE THREAD

**Issue by maaeli** [maaeli/freesas#5]
for auto_gpa: max of Iq mis-guessed
for autoRg: probably Imax too high

**Comment by maaeli** [comment]
Ideas:
- Just try a second time from script, discarding first N points 
- In function, take position of maximum into account and try a second time
- Include indices in error, to allow caller an educated reaction

**Comment by kif** [comment]
Indeed, for Rg, one could start by keeping only the largest section with strictly positive intensities.
Having a very large I then negative values creates an issue.
